cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/TableId.java

```diff
@@ -45,19 +46,34 @@ public String getTablePrefix() {
     return namespace;
   }
 
+  @VisibleForTesting
+  public String getHBaseNamespace() {
```

Comment:
I would rather not have this method. For testing, you can simply call `HBaseTableUtil.toBaseNamespace(id.getNamespace())`